modules/developer_manual/pages/_partials/webdav_api/search/file_properties.adoc

```diff
@@ -9,7 +9,7 @@
 |Namespace
 
 |getcontentlength
-|The size of the response body, _in bytes_, sent to the recipient.
+|The file's content length.
```

Comment:
this property is only returned for **files** and is absent (404) for folders/collections

for folders, the property "size" (which is present below) is used instead of "getcontentlength".
for files, the property "size" is absent (404)